pkg/provider/bitbucketdatacenter/bitbucketdatacenter.go

```diff
@@ -371,79 +372,82 @@ func (v *Provider) GetConfig() *info.ProviderConfig {
 }
 
 func (v *Provider) GetFiles(ctx context.Context, runevent *info.Event) (changedfiles.ChangedFiles, error) {
-	OrgAndRepo := fmt.Sprintf("%s/%s", runevent.Organization, runevent.Repository)
-	if runevent.TriggerTarget == triggertype.PullRequest {
-		opts := &scm.ListOptions{Page: 1, Size: apiResponseLimit}
-		changedFiles := changedfiles.ChangedFiles{}
-		for {
-			changes, _, err := v.Client().PullRequests.ListChanges(ctx, OrgAndRepo, runevent.PullRequestNumber, opts)
-			if err != nil {
-				return changedfiles.ChangedFiles{}, fmt.Errorf("failed to list changes for pull request: %w", err)
-			}
-
-			for _, c := range changes {
-				changedFiles.All = append(changedFiles.All, c.Path)
-				if c.Added {
-					changedFiles.Added = append(changedFiles.Added, c.Path)
-				}
-				if c.Modified {
-					changedFiles.Modified = append(changedFiles.Modified, c.Path)
-				}
-				if c.Renamed {
-					changedFiles.Renamed = append(changedFiles.Renamed, c.Path)
+	if v.changedFiles == nil {
+		OrgAndRepo := fmt.Sprintf("%s/%s", runevent.Organization, runevent.Repository)
```

Comment:
```suggestion
		orgAndRepo := fmt.Sprintf("%s/%s", runevent.Organization, runevent.Repository)
```

[super nit] It bugs me that we have "titled" variable with uppercase but that may be me writting too much python 😅

Reply:
This was just a whitespace change but I agree, personally not a fan of PascalCase (especially when it implies language semantics sometimes). Renamed to camelCase